CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**3.0.16 - 10/31/24**
```

Comment:
is this intended to be tomorrow?

Reply:
Yes just b/c I rarely get approvals this late in the day